src/segment-loader.js

```diff
@@ -666,6 +688,33 @@ export default class SegmentLoader extends videojs.EventTarget {
       return null;
     }
 
+    let bufferedBytes = this.maxBufferedBytes();
```

Comment:
Since the only byte information we have is at the entire segment level, if our buffer size ends up not locked to segment boundaries, it's unknown how much data we should include from that partial segment. So we have `minBufferedBytes` which excludes it (and gives us a lower bound), and `maxBufferedBytes` which includes it (and gives us an upper bound).

So here, we're getting the upper bound for the amount of bytes in the buffer.

Reply:
> So in the above we're saying "I'm guessing we were able to fit the whole last segment in the buffer"(bufferedBytes) "now see if we can fit the next one in without going over the maxBytes"?

I'm not sure what this has to do with the last segment...? What happened in the past is irrelevant, it's strictly based on the amount of bytes currently in the buffer. We're just testing if the next segment request (`nextSegmentSize`) will push the (worst case) amount of bytes in the buffer (`bufferedBytes`) over the maximum amount of bytes allowed in the buffer (`maxBytes`), which was calculated from the amount of bytes in the buffer when the `QuotaExceededError` happened.

Reply:
Oh, I only said the last segment since we're appending full segments in here and we're using segment level info to guess how much we have in the buffer(referencing your explanation of `minBufferedBytes` and `maxBufferedBytes`).

Still, it would be useful to have a comment over the declaration of `bufferedBytes` explaining that "we're assuming the worst case amount of bytes in the buffer"